spec/loophole-spec.coffee

```diff
@@ -28,7 +28,47 @@ describe "Loophole", ->
       result = allowUnsafeNewFunction -> 42
       expect(result).toBe 42
 
+    it "works with async callback", ->
+      result = null
+
+      runs ->
+        allowUnsafeNewFunction (callback) ->
+          setTimeout(
+            ->
+              result = 42
+              callback()
+            100
+          )
+
+      waitsFor(
```

Comment:
I don't understand this. Is it a coffeeScript thing?

from jasmine doc
```js
waitsFor(function() {
      value++;
      return flag;
}, "The Value should be incremented", 750);
```
The signature is in this order.

BTW we should really use jasmine 2.X, `callback` api makes more sense.